merlin/models/tf/inputs/embedding.py

```diff
@@ -335,120 +343,115 @@ def get_config(self):
 
 def Embeddings(
     schema: Schema,
-    pre: Optional[BlockType] = None,
-    post: Optional[BlockType] = None,
-    aggregation: Optional[TabularAggregationType] = None,
-    block_name: str = "embeddings",
+    embedding_dims: Optional[Union[Dict[str, int], int]] = None,
+    infer_dim_fn: Callable[[ColumnSchema], int] = infer_embedding_dim,
+    trainable: Optional[Dict[str, bool]] = None,
     sequence_combiner: Optional[
         Union[str, tf.keras.layers.Layer, Dict[str, Union[str, tf.keras.layers.Layer]]]
     ] = "mean",
-    embedding_dims: Optional[Dict[str, int]] = None,
-    embedding_dim_default: Optional[int] = 32,
-    infer_embedding_sizes: bool = True,
-    infer_embedding_sizes_multiplier: float = 2.0,
-    infer_embeddings_ensure_dim_multiple_of_8: bool = True,
-    trainable: Optional[Dict[str, bool]] = None,
     embeddings_initializers: Optional[
-        Union[Dict[str, Callable[[Any], None]], Callable[[Any], None]]
+        Union[Dict[str, Callable[[Any], None]], Callable[[Any], None], str]
```

Comment:
I think the type of of the initializers in `EmbeddingOptions` needs to be updated to match this for consistency.